src/test/java/uk/gov/pay/adminusers/utils/email/EmailValidatorIsPublicSectorEmailTest.java

```diff
@@ -159,7 +161,7 @@ static Collection<Object[]> data() {
                 {"valid@subdomain.uksbs.co.uk", true},
                 {"valid@subdomain.visitbritain.org", true},
                 {"valid@subdomain.wmca.org.uk", true},
-                {"valid@subdomain.york.ac.uk", true},
+                {"valid@subdomain.york.ac.uk", true}
```

Comment:
This is fine, but I am wondering if we should prefer lists to be comma terminated, is this in our styleguide? I'll approve otherwise. :)